module-2/lab-probability-distribution/your_code/main.ipynb

```diff
@@ -12,11 +12,36 @@
   },
```

Comment:
- Sometimes it's useful to seed the random number generator. Seeding a pseudo-random number generator gives it its first "previous" value. Each seed value will correspond to a sequence of generated values for a given random number generator. That is, if you provide the same seed twice, you get the same sequence of numbers twice. https://machinelearningmastery.com/how-to-generate-random-numbers-in-python/
- What about "Compare the distributions above. What do you observe? Explain with the Central Limit Theorem."? Comments on your findings are not available.
- This is not quite correct: "La variación de la función de distribución binomial es 5 veces mayor de 1 a o 2 que de 0 a 1".  Could be something like, "a medida que aumenta el numero de visualizadores, la probabilidad de click es menor" ?
- What about "What did you notice from the distribution plots? Comment your findings"? Comments on your findings are not available.
- Normality tests are quite sensitive to sample size thus for a sample size below a couple of thousands it should be a better idea to evaluate normality through a plot, in particular a quantile-quantile plot (QQ plot). A Q-Q plot is a scatterplot created by plotting two sets of quantiles against one another. It plots the quantiles of our sample vs the theoretical quantiles of the distribution type we want to test (this case a normal distribution) How to do this in Python? https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.probplot.html.
- There are 2 other normality tests (additional to D'Agostino k^2 test, normaltest in Python) that are very used in to test normality of our data. These are Kolmogorov-Smirnof and Shapiro-Wilk tests. I encourage you to deepen on these type of tests. In case you are interested, I can provide you some useful resources.
- Interpretation of the CLT: take a look on these 2 articles (the most clear explanations I've found). You can replicate in Python some of the examples they include.
https://towardsdatascience.com/understanding-the-central-limit-theorem-642473c63ad8.
https://www.statisticshowto.datasciencecentral.com/probability-and-statistics/normal-distributions/central-limit-theorem-definition-examples/
- Normal distribution can be used as an approximation of Binomial and Poisson distributions under certain circumstances. Generally speaking:
Binomial: X ~ B(n, p) and if n is large and/or p is close to ½, then X is approximately N(np, npq)
Poisson: X ~ Po(l) then for large values of l, X ~ N(l, l) approximately.
However, we must be careful because binomial and poisson distributions are discrete while Normal distribution is continuous. We must use a continuity correction.
-Poisson: expresses the probability of a given number of events occurring in a fixed interval of time or space if these events occur with a known constant rate and independently of the time since the last event.